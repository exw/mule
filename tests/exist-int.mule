4 : exist t. t
