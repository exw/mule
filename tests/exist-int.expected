
Desugared:

	(App ((WithType (Quant (() Exist t (Var (() t))))) (Const (Integer 4))))

inferred type:

	(Quant (() Exist a (Var (() a))))

Runtime term:

	(Integer 4)

Evaluated:

	(Integer 4)
